sdk/openai/Azure.AI.OpenAI/src/Custom/AudioTranscription.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
```

Comment:
Remove "\<auto-generated/\>" comment.

Reply:
Thanks; I did a full pass across all instances in /custom/.